components/dashboard/src/teams/TeamSettings.tsx

```diff
@@ -185,3 +165,91 @@ export default function TeamSettingsPage() {
+    const updateTeamSettings = useUpdateOrgSettingsMutation();
+    const [defaultWorkspaceImage, setDefaultWorkspaceImage] = useState(settings?.defaultWorkspaceImage ?? "");
+    const { toast } = useToast();
+
+    useEffect(() => {
+        if (!settings) {
+            return;
+        }
+        setDefaultWorkspaceImage(settings.defaultWorkspaceImage ?? "");
+    }, [settings]);
+
+    const handleUpdateTeamSettings = useCallback(
+        async (newSettings: Partial<OrganizationSettings>) => {
+            if (!org?.id) {
+                throw new Error("no organization selected");
+            }
+            if (!org.isOwner) {
+                throw new Error("no organization settings change permission");
+            }
+            try {
+                await updateTeamSettings.mutateAsync({
+                    ...settings,
+                    ...newSettings,
+                });
+                if (newSettings.defaultWorkspaceImage) {
+                    toast("Default workspace image has been updated.");
+                }
+            } catch (error) {
+                console.error(error);
+                toast(
+                    error.message
+                        ? "Failed to update organization settings: " + error.message
+                        : "Oh no, there was a problem with our service.",
+                );
+            }
+        },
+        [updateTeamSettings, org?.id, org?.isOwner, settings, toast],
+    );
+
+    return (
+        <form
+            onSubmit={(e) => {
+                e.preventDefault();
+                handleUpdateTeamSettings({ defaultWorkspaceImage });
+            }}
+        >
+            <Heading2 className="pt-12">Collaboration & Sharing</Heading2>
+            <Subheading className="max-w-2xl">
+                Choose which workspace images you want to use for your workspaces.
+            </Subheading>
+
+            {updateTeamSettings.isError && (
+                <Alert type="error" closable={true} className="mb-2 max-w-xl rounded-md">
+                    <span>Failed to update organization settings: </span>
+                    <span>{updateTeamSettings.error.message || "unknown error"}</span>
+                </Alert>
+            )}
+
+            <CheckboxInputField
+                label="Workspace Sharing"
+                hint="Allow workspaces created within an Organization to share the workspace with any authenticated user."
+                checked={!settings?.workspaceSharingDisabled}
+                onChange={(checked) => handleUpdateTeamSettings({ workspaceSharingDisabled: !checked })}
+                disabled={isLoading || !org?.isOwner}
+            />
+
+            <Heading2 className="pt-12">Workspace Settings</Heading2>
+            <TextInputField
+                label="Default Image"
+                // TODO: Provide document links
+                hint="Use any official Gitpod Docker image, or a URL to an image."
```

Comment:
really nice work so far! Here's one little nit I had
```suggestion
                hint="Use any official Gitpod Docker image, or Docker image reference"
```